src/components/styledSelectOption/styledSelectOption.tsx

```diff
@@ -1,5 +1,11 @@
-import { type KeyboardEventHandler, type MouseEventHandler } from 'react'
+import {
+  useRef,
+  type KeyboardEventHandler,
+  type MouseEventHandler,
+} from 'react'
 import classNames, { type Argument } from 'classnames'
+import { measureText } from '../../utils/measureText'
```

Comment:
If I was going to do this over I'd change it to a default export but it doesn't matter enough to make the change in all three places it's imported lol.